packages/trace-viewer/src/ui/defaultSettingsView.tsx

```diff
@@ -37,7 +38,12 @@ export const DefaultSettingsView: React.FC<{}> = () => {
           value: shouldPopulateCanvasFromScreenshot,
           set: setShouldPopulateCanvasFromScreenshot,
           name: 'Display canvas content',
-          title: 'Attempt to display the captured canvas appearance in the snapshot preview. May not be accurate.'
+          title: 'Attempt to display the captured canvas appearance in the snapshot preview. May not be accurate.',
+        },
+        {
+          value: showAllActions,
+          set: setShowAllActions,
+          name: 'Show all actions',
```

Comment:
Add a `title` with a more complete description?

Reply:
I will in the next PR.